### PR TITLE
Add Italian Subtitles

### PR DESCRIPTION
**Pull request by hrumag**
Added files (.md and .sbv):

- Stealth Addresses
- Ring Signatures
- RingCT
- Kovri

@erciccione please give 'em a review.

Thanks,
hrumag

### CODE REVIEW

File: translations/it/kovri.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+La rete Tor possiede Autorità di Directory "semi-fidate" che conferiscono ad un piccolo gruppo di operatori di nodi Tor un'influenza superiore sul consensus della rete.
+Il consensus della rete determina sostanzialmente chi ha il permesso di inoltrare il traffico sulla rete Tor basandosi sulla visione di suddette Autorità.
+Inoltre, attacchi basati su correlazione sono possibili su VPN fidate, rendendo, per attaccanti di rilievo, molto semplice la de-anonimizzazione del traffico degli utenti.
+Quindi cosa possono fare Alice, Bob, Charlie e Dave per rispondere a queste minacce?
+Possono usare Kovri!
+Se usano esclusivamente Kovri per connettersi alla rete Monero, nessuno conoscerà mai i loro indirizzi IP, rendendo quindi inattuabile la sorveglianza passiva, e allo stesso tempo migliorando in modo sostanziale la resistenza di Monero alla censura.
+Come potete vedere, Kovri è un software che consente agli utenti di inviare transazioni in modo ancora più privato.
+Addentriamoci ora in alcuni dettagli tecnici per avere una migliore comprensione su come Kovri lavora al suo interno.
+
+## 5) Kovri e gli Attributi Tecnici
```
Comment (serhack) [inline translations/it/kovri.md]: 5) Kovri e le Specifiche Tecniche

File: translations/it/kovri.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Per rafforzare ulteriormente la privacy, pagando un piccolo prezzo in termini di performance, gli utenti possono connettersi a più nodi.
+Essenzialmente, Kovri copre il traffico internet di un'applicazione per renderlo anonimo all'interno della rete.
+Grazie a queste caratteristiche, Kovri è un'ottima soluzione per comunicare su IRC, via email, o per accedere a servizi nascosti.
+
+## 6) - Conclusioni
+
+Kovri rivoluzionerà le connessioni peer-to-peer ed aumenterà globalmente la resilienza e la privacy della rete.
+Nodi malevoli non potranno più minacciare gli utenti che creano transazioni né bloccare la trasmissione delle transazioni sulla rete.
+Kovri verrà reso disponibile all'interno dei futuri rilasci di Monero e sarà abilitato di default.
+Inoltre, Kovri esporrà delle API che ne consentiranno l'utilizzo anche ad altre criptovalute ed applicazioni, non solo a Monero.
```
Comment (serhack) [inline translations/it/kovri.md]: esporrà -> implementerà

**Review by serhack — COMMENTED** [review]
Good job

File: translations/it/kovri.md (hunk 1 above)
Comment (hrumag) [reply translations/it/kovri.md]: D:\Monero\hrumag\promo-video\translations\it\kovri.md (1 hit)
	Line 48: ## 5) Kovri e gli Attributi Tecnici
  D:\Monero\hrumag\promo-video\translations\it\ringct.md (2 hits)
	Line 25: ## 4)  Attributi non tecnici delle RingCT
	Line 34: ## 5) – RingCT e Attributi Tecnici

Should I replace here too?

How could I translate "Non technical attributes"? "Specifiche non tecniche"?. It sounds a little bit weird.
Could we find another solution if you don't like "attributi (non) tecnici"?

"Caratteristiche (non) tecniche" for example?

File: translations/it/kovri.md (hunk 2 above)
Comment (hrumag) [reply translations/it/kovri.md]: Agreed,

File: translations/it/kovri.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+## 1) - Intro
+
+Monero è una moneta elettronica, sicura e non tracciabile. E' opensource, decentralizzata e liberamente accessible a tutti.
+In questo video ci concentreremo su una tecnologia per l'anonimato chiamata Kovri.
```
Comment (erciccione) [inline translations/it/kovri.md]: I think `tecnologia di anonimizzazione` suits better

File: translations/it/kovri.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+Finora abbiamo spiegato come Monero offusca le informazioni registrate sulla blockchain.
+Le firme ad anello oscurano il mittente.
+Gli indirizzi nascosti garantiscono che osservatori esterni non possano conoscere l'indirizzo di destinazione.
+Le RingCT nascondono l'ammontare del Moneroj che vengono trasferiti.
+Eppure sulla rete possono trapelare alcune informazioni personali quando viene effettuata una transazione.
+Questo problema viene affrontato da Kovri.
+
+## 3) - La terminologia - Kovri
+
+Kovri è una tecnologia per l'anonimato gratuita, decentralizzata e basata sulle specifiche aperte di I2P.
```
Comment (erciccione) [inline translations/it/kovri.md]: `Tecnologia di anonimizzazione`

File: translations/it/kovri.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## 4) Esempi
+
+Consideriamo alcuni scenari per vedere come questo software leggero e focalizzato sulla privacy aiuta a rafforzare la privacy dell'utente.
+Supponiamo che Alice voglia inviare dei Moneroj a Bob.
+Il portafoglio di Alice crea una transazione e la trasmette alla rete Monero.
+La rete Monero è formata da nodi che comunicano fra loro trasmettendosi messaggi ed utilizzando i propri indirizzi IP.
+Questo significa che è possibile tracciare i dati a livello geografico mentre essi transitano su internet, dal punto iniziale al punto finale passando per tutti i punti intermedi.
+Anche se gli indirizzi di portafoglio del mittente e del destinatario - come anche l'ammontare dei Moneroj trasferiti - rimangono privati, Alice corre un rischio quando trasmette la sua transazione poiché alcuni nodi potrebbero registrare gli indirizzi IP.
+Un avversario con risorse sufficienti potrebbe tentare di associare le transazioni agli indirizzi IP per determinare il luogo da cui sono state generate.
```
Comment (erciccione) [inline translations/it/kovri.md]: `avversario` -> `soggetto malizioso` or `soggetto malevolo` or `malintenzionato`?

File: translations/it/kovri.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+Questo tentativo potrebbe portare l'avversario a non ritrasmettere la transazione di Alice al resto della rete; o, ancor peggio, a presentarsi alla porta di Alice.
```
Comment (erciccione) [inline translations/it/kovri.md]: same as above for `avversario`

File: translations/it/kovri.md (hunk 1 above)
Comment (erciccione) [reply translations/it/kovri.md]: I like "Caratteristiche tecniche" and "caratteristiche non tecniche"

File: translations/it/kovri.sbv [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+- come anche l'ammontare dei Moneroj trasferiti - rimangono privati,
+
+0:01:36.060,0:01:39.040
+Alice corre un rischio quando trasmette la sua transazione
+
+0:01:39.040,0:01:42.100
+poiché alcuni nodi potrebbero registrare gli indirizzi IP.
+
+0:01:42.100,0:01:43.940
+Un avversario con risorse sufficienti
```
Comment (erciccione) [inline translations/it/kovri.sbv]: See suggestions above about `avversario`

File: translations/it/kovri.sbv [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+
+0:01:43.940,0:01:47.140
+potrebbe tentare di associare le transazioni agli indirizzi IP
+
+0:01:47.140,0:01:49.920
+per determinare il luogo da cui sono state generate.
+
+0:01:49.920,0:01:55.040
+Questo tentativo potrebbe portare l'avversario a non ritrasmettere la transazione di Alice al resto della rete;
```
Comment (erciccione) [inline translations/it/kovri.sbv]: See comment above for `avversario`

File: translations/it/ring_signatures.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+## 1) - Intro
+
+Monero è una moneta elettronica, sicura e non tracciabile. E' opensource, decentralizzata e liberamente accessible a tutti.
+In questo video, ci concentreremo sulle firme ad anello.
+ 
+## 2) - Ripasso sulla tecnologia
+ 
+Nel video precedente abbiamo spiegato come gli indirizzi nascosti Monero assicurano che gli output non possano essere associati all'indirizzo pubblico del destinatario.
+Questo è possible grazie all'utilizzo di chiavi pubbliche monouso di destinazione.
```
Comment (erciccione) [inline translations/it/ring_signatures.md]: `chiavi di destinazioni pubbliche monouso`

File: translations/it/ring_signatures.sbv [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+Nel video precedente
+
+0:00:14.580,0:00:17.760
+abbiamo spiegato come gli indirizzi nascosti Monero assicurano che gli output
+
+0:00:17.760,0:00:20.560
+non possano essere associati all'indirizzo pubblico del destinatario.
+
+0:00:20.560,0:00:24.820
+Questo è possible grazie all'utilizzo di chiavi pubbliche monouso di destinazione.
```
Comment (erciccione) [inline translations/it/ring_signatures.sbv]: see comment above

File: translations/it/ringct.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+Nonostante gli "impegni" ("commitments") appaiano come numeri casuali, i minatori sono ancora in grado di confermare che l'ammontare di Moneroj inviati a Bob è lo stesso dell'ammontare di fondi disponibile. [Vedi Nota 1]
+Un altro aspetto importante di una transazione RingCT è la "prova di intervallo" ("range proof"), che impedisce ai mittenti di prendere un impegno su valori negativi. In questo modo la fornitura di Moneroj viene resa sicura.
+Una prova di intervallo prova crittograficamente che gli ammontare di una transazione sono maggiori di zero e minori di un numero arbitrario.
+Sebbene un osservatore esterno non sia in grado di vedere i veri ammontare negli output di una transazione, può comunque confermare che la transazione è legittima e quindi accettabile dalla rete.
+
+## 6) - Conclusioni
+
+Grazie alle funzionalità orientate alla privacy offerte da Monero, gli utenti possono trasferire Moneroj a chiunque vogliono; nessuno saprà mai quanti Moneroj sono stati inviati, da chi sono stati inviati e a chi sono stati inviati.
```
Comment (erciccione) [inline translations/it/ringct.md]: `chiunque vogliano`

File: translations/it/ringct.sbv [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+può comunque confermare che la transazione è legittima
+
+0:03:53.000,0:03:55.140
+e quindi accettabile dalla rete.
+
+0:03:55.140,0:03:58.040
+Grazie alle funzionalità orientate alla privacy offerte da Monero,
+
+0:03:58.040,0:04:01.120
+gli utenti possono trasferire Moneroj a chiunque vogliono;
```
Comment (erciccione) [inline translations/it/ringct.sbv]: `chiunque vogliano`

**Review by erciccione — COMMENTED** [review]
Thanks @hrumag . I proposed some small changes, but very good overall

File: translations/it/kovri.md (hunk 1 above)
Comment (hrumag) [reply translations/it/kovri.md]: Agreed,

File: translations/it/kovri.md (hunk 3 above)
Comment (hrumag) [reply translations/it/kovri.md]: Agreed.

File: translations/it/kovri.md (hunk 4 above)
Comment (hrumag) [reply translations/it/kovri.md]: Agreed.

File: translations/it/kovri.md (hunk 5 above)
Comment (hrumag) [reply translations/it/kovri.md]: "Soggetto malevolo". Agreed.

File: translations/it/kovri.md (hunk 6 above)
Comment (hrumag) [reply translations/it/kovri.md]: Agreed.

File: translations/it/kovri.sbv (hunk 7 above)
Comment (hrumag) [reply translations/it/kovri.sbv]: Agreed.

File: translations/it/kovri.sbv (hunk 8 above)
Comment (hrumag) [reply translations/it/kovri.sbv]: Agreed.

File: translations/it/ring_signatures.md (hunk 9 above)
Comment (hrumag) [reply translations/it/ring_signatures.md]: Agreed.

File: translations/it/ring_signatures.sbv (hunk 10 above)
Comment (hrumag) [reply translations/it/ring_signatures.sbv]: Agreed.

File: translations/it/ringct.md (hunk 11 above)
Comment (hrumag) [reply translations/it/ringct.md]: Agreed.

File: translations/it/ringct.sbv (hunk 12 above)
Comment (hrumag) [reply translations/it/ringct.sbv]: Agreed.

**Review by erciccione — APPROVED** [review]

**Review by serhack — APPROVED** [review]

**Comment by hrumag** [comment]
@ajs-xmr Ready to merge. :)

hrumag

**Comment by hrumag** [comment]
Hi @ajs-xmr, today I checked on Youtube and Italian subtitles are not available yet.
Is there an ETA for that? I would like to show the videos to my parents but they do not speak English. 🤣 
Thanks.

EDIT: I loaded the subtitles on Youtube as a contribution. Now they have to be approved by the video owner.

hrumag

**Comment by ajs-xmr** [comment]
@savandra ^

**Pull request merged**